dandi/cli/cmd_validate.py

```diff
@@ -36,19 +36,40 @@ def validate_bids(
         dandi validate-bids /my/path
     """
 
-    from ..bids_utils import is_valid, report_errors
+    from ..validate import Severity
     from ..validate import validate_bids as validate_bids_
 
-    validator_result = validate_bids_(
+    validator_result = validate_bids_(  # Controller
         *paths,
         report=report,
         report_path=report_path,
         schema_version=schema,
     )
-    valid = is_valid(validator_result)
-    report_errors(validator_result)
 
-    if not valid:
+    for i in validator_result:
+        if i.path:
+            scope = i.path
+        elif i.path_regex:
+            scope = i.path_regex
+        else:
+            scope = i.dataset_path
+        echo_string = f"[{i.id}] {scope}: {i.message}"
+        if i.severity == Severity.ERROR:
+            click.secho(
+                echo_string,
+                bold=True,
+                fg="red",
+            )
+        if i.severity == Severity.WARNING:
```

Comment:
at least
```suggestion
        elif i.severity == Severity.WARNING:
```
for clarity.  In general, I would have even folded this into a single `.secho` with just changing color based on severity value to avoid duplication, but since such a minor -- not worth RFing for that

Reply:
[48de132](https://github.com/dandi/dandi-cli/pull/1104/commits/48de132d361d3da53b1b1775b3721a1f1d61e92b)